src/Type.js

```diff
@@ -104,6 +105,15 @@ export default class Type {
 		return ret;
 	}
 
+	/**
+	 * Returns a percentage range for values of this type
+	 * @param {number} scale
+	 */
+	pctRange (scale = 1) {
```

Comment:
I’d rather avoid abbreviations like "pct".
```suggestion
	percentageRange (scale = 1) {
```